lib/cadet_web/controllers/module_controller.ex

```diff
@@ -0,0 +1,46 @@
+defmodule CadetWeb.ModuleController do
+    @moduledoc """
+    Synchronize local module files witn GitHub repository via WebHook
+    and repond HTTP resquest from frontend
+    """
+  
+    use CadetWeb, :controller
+  
+    use PhoenixSwagger
+  
+    def handle(conn, _payload) do
+      repo = Git.new("priv/module_static")
```

Comment:
wondering about the choice of name for the folder. This is the destination of the webhook, right? So priv/module_static gets populated by the modules from the modules repo, right? In that case, what is "static" about it?

Reply:
Okay, we can change the name of the destination folder, what about putting the module files under priv/modules?